### PR TITLE
Updated TinyWebDB files

### PR DESCRIPTION
**Pull request by Kanishka-Developer**
These files are updated for the latest changes to Google Cloud Platform.
These are not to be merged - these files are meant to _replace_ the existing files.
I also edited the index.html file to be more Makeroid-ish.
For any queries or whatever, contact me by email. (kanishka.developer@gmail.com)

### CODE REVIEW

**Comment by pavi2410** [comment]
I just started to work on it, thinking of using Flask. I have already done many changes which will create merge conflicts.

My changes are here: https://github.com/pavi2410/TinyWebDB

**Comment by Kanishka-Developer** [comment]
@pavi2410 The only files I actually edited are app.yaml and index.html.
Can you pull index.html (along with the images)?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# Makeroid TinyWebDB Server
-The official Makeroid TinyWebDB Server.
+# TinyWebDB (Updated)
```
Comment (barreeeiroo) [inline README.md]: Remove the (Updated) here and set `Makeroid TinyWebDB`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
+Updated TinyWebDB service for Makeroid
```
Comment (barreeeiroo) [inline README.md]: Here remove the `Updated `

**Review by barreeeiroo — CHANGES_REQUESTED** [review]

**Comment by Kanishka-Developer** [comment]
Done, @barreeeiroo.

**Comment by Kanishka-Developer** [comment]
@pavi2410 @barreeeiroo Anything else?

**Review by barreeeiroo — APPROVED** [review]

**Pull request merged**